tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -875,10 +876,8 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
       // need to check the input edges.
       [](const Edge* edge) { return true; }, OutputEdgeValidator(),
       segment_options, &initial_segments));
-  if (initial_segments.size() > 1) {
-    VLOG(0) << "MULTIPLE tensorrt candidate conversion: "
-            << initial_segments.size();
-  }
+  VLOG(0) << "Number of TensorRT candidate segments: "
+    << initial_segments.size();
```

Comment:
Please use LOG(INFO), and fix the alignment.

Reply:
Done